module/zcommon/zfeature_common.c

```diff
@@ -439,9 +439,21 @@ zpool_feature_init(void)
 	    ZFEATURE_TYPE_BOOLEAN, userobj_accounting_deps);
 	}
 
+	{
+	static const spa_feature_t bookmark_v2_deps[] = {
+		SPA_FEATURE_EXTENSIBLE_DATASET,
```

Comment:
Should SPA_FEATURE_BOOKMARKS be in this list?

Reply:
Wow. good catch. Fixed.